src/python/pants/backend/python/subsystems/python_native_code.py

```diff
@@ -10,7 +10,7 @@
 
 
 class PythonNativeCode(Subsystem):
-    """A subsystem which exposes components of the native backend to the python backend."""
+    """Options for building native code using Python, e.g. when resolving distributions."""
```

Comment:
This subsystem should be rolled into `subprocess-environment`, but we missed the boat with 2.0.0.